packages/sitecore-jss-react/src/index.ts

```diff
@@ -100,3 +100,9 @@ export { withPlaceholder } from './enhancers/withPlaceholder';
 export { withDatasourceCheck } from './enhancers/withDatasourceCheck';
 export { EditFrameProps, EditFrame } from './components/EditFrame';
 export { ComponentBuilder, ComponentBuilderConfig } from './ComponentBuilder';
+export {
+  FieldMetadata,
+  FieldMetadataComponent,
+  FieldMetadataComponentProps,
```

Comment:
I'm wondering if we even need to expose _metadata_ related types. Essentially, they are our internals and shouldn't be used by customer directly.
We just need to export _getFieldMetadataMarkup_ if it's possible